go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/karmada-io/karmada
 
-go 1.22.7 // keep in sync with .go-version, Readme.md#Prerequisites, hack/util.sh
+go 1.22.11 // keep in sync with .go-version and hack/util.sh
```

Comment:
This change is simply cherry-picked. But looks like release-1.11 has more things keep in sync. Do I need to update `Readme.mc#Reprequisites` and also keep this note in this comment?

Reply:
Your carefulness presses me!
Thanks, but it seems the `Readme.mc#Reprequisites` was already updated on release-1.11, but the comment was left unaltered. So, this change is perfect!